examples/app_demo.py

```diff
@@ -7,16 +7,21 @@
 
```

Comment:
This config still can't download for me.

![image](https://github.com/user-attachments/assets/1bb8c555-4efd-4eec-a4b8-e35cc50a0125)

Reply:
Fixed.